src/libPMacc/include/algorithms/math/floatMath/trigo.tpp

```diff
@@ -47,6 +47,21 @@ struct Sin<float>
 };
 
 template<>
+struct ASin<float>
+{
+    typedef float result;
+
+    HDINLINE float operator( )(const float& value )
```

Comment:
Space is missing after the parentheses. `HDINLINE float operator( )( const....)`